tests/conftest.py

```diff
@@ -360,7 +360,6 @@ def project_factory(
     repo: TestRepository,
     installed: Repository,
     default_python: str,
-    load_required_fixtures: None,
```

Comment:
Is this really unused? Then, we don't need `load_required_fixtures()` at all. Although, `required_fixtures()` returns an empty list here, it's overridden in `test_add.py` for example. Apparently, the tests don't fail, so maybe we can just remove all of this?

Reply:
On the other hand it might be dangerous to remove this because it was introduced in  #5745. Thus, a passing test is not prove enough because it probably avoids race conditions.

Reply:
I tried removing more than this and tests for the add and show commands started in fact breaking. I didn't experience any race condition while removing this line though and re-running the whole suite several times. But if you think it's best to keep it I won't complain about an extra line, just say so :)

Reply:
Ah, `load_required_fixtures()` is marked with `autouse=True`, which means this change makes sense. 😀